stress/src/user_events.rs

```diff
@@ -11,11 +11,11 @@
 //!
 // Conf - AMD EPYC 7763 64-Core Processor 2.44 GHz, 64GB RAM, Cores:8 , Logical processors: 16
 // Stress Test Results (user_events disabled)
-// Threads: 1 - Average Throughput: 42,086,520 iterations/sec
-// Threads: 5 - Average Throughput: 35,767,375 iterations/sec
-// Threads: 10 - Average Throughput: 29,189,340 iterations/sec
-// Threads: 16 - Average Throughput: 19,579,138 iterations/sec
-
+// Threads: 1 - Average Throughput: 43 iterations/sec
```

Comment:
43 looks too low for throughput even for single thread.

Reply:
its 43M. I missed writing M there !